doc/resources/assets/hexcasting/lang/en_us.flatten.json5

```diff
@@ -43,6 +43,11 @@
       hide: "Click to hide recipes",
     },
 
+    villagerrequirements: {
+      any: "Villager must be a {} or higher",
+      professional: "Villager must be a {} {} or higher",
```

Comment:
these should have names, like `Villager must be a {level} {profession} or higher`. then in the code you'd do `.format(level=..., profession=...)`